coverage.sh

```diff
@@ -32,6 +32,15 @@ fi
 ./sizes
 ./constants
 ./multi
+for i in $(./hashsum | tail -n +3 | awk '{print $1}' | sort); do echo -n "$i: " && ./hashsum $i testprof/test.key ; done > hashsum_tv.txt
```

Comment:
Wouldn't it be better to move this part to `build.sh` - section `echo -n "testing..."` where we already do similar diffs with `tv_gen` outputs?